src/Service/Revision/RevisionService.php

```diff
@@ -173,6 +173,22 @@ public function create(ContentType $contentType, ?UuidInterface $uuid = null, ar
         return $this->dataService->newDocument($contentType, null === $uuid ? null : $uuid->toString(), $rawData);
     }
 
+    /**
+     * @param ?array<mixed> $mergeRawData
+     */
+    public function copy(Revision $revision, ?array $mergeRawData = null): void
+    {
+        $copiedRevision = $revision->clone();
+
+        if ($mergeRawData) {
+            $copiedRevision->setRawData(\array_merge($copiedRevision->getRawData(), $mergeRawData));
+        }
+
+        $form = null;
```

Comment:
Why not pass null directly to the method below, instead of instantiating a variable ?

Reply:
Aah go reviewing, because the finalizeDraft has &$form as input argument. It will be refactored in the future. For now it's to make php happy.

Reply:
fair enough